agent/handlers/task_server_setup.go

```diff
@@ -18,6 +18,8 @@ import (
 	"strconv"
 	"time"
 
+	v4 "github.com/aws/amazon-ecs-agent/agent/handlers/v4"
```

Comment:
nit: group this import to below, same for another file

Reply:
sure, change it